settings/security/third-party.rst

```diff
@@ -0,0 +1,50 @@
+Third-Party Applications
+************************
+
+Third party authentication is a great way to help your users to login to Zammad
+easier. If the account is yet unknown, Zammad will create a new user
+automatically, without the user needed to interact (e.g. type in his name).
+Another big advantage of this feature is that your user doesn't need another
+password to remember.
+
+.. toctree::
+   :caption: Login Providers
+   :maxdepth: 1
+
+   third-party/facebook
+   third-party/github
+   third-party/gitlab
+   third-party/google
+   third-party/microsoft
+   third-party/twitter
+   third-party/saml
+
+.. note:: 
+
+   We're currently missing documentation for the following login providers:
+
+      * LinkedIn
+      * Weibo
+
+.. _automatic-account-linking:
+
+Automatic account link on initial logon
```

Comment:
logon oder login?

Reply:
The setting contains logon:
![image](https://user-images.githubusercontent.com/6549061/129939328-b73d215c-fa43-484b-9b77-766ac93742d2.png)